From 192e2e4f342281a066ccee8f2686e052ceea4698 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Fri, 6 Aug 2021 19:10:45 +0900
Subject: [PATCH 74/99] ODROID-N2: arm64/dts: pwmfan: add new temp to trigger

This device overlay patch is to set lower temperature that triggers
cooler fan at 45 degress celsius rather than 65 degrees celsius.

Signed-off-by: Dongjin Kim <tobetter@gmail.com>
Change-Id: Ifb1cfa5a06b2c7b25f9e14be5c9b800e16e10ce2
---
 .../dts/amlogic/overlays/odroidn2/Makefile    |  1 +
 .../overlays/odroidn2/pwmfan-lowtemp.dts      | 20 +++++++++++++++++++
 2 files changed, 21 insertions(+)
 create mode 100644 arch/arm64/boot/dts/amlogic/overlays/odroidn2/pwmfan-lowtemp.dts

diff --git a/arch/arm64/boot/dts/amlogic/overlays/odroidn2/Makefile b/arch/arm64/boot/dts/amlogic/overlays/odroidn2/Makefile
index 70ea1faad2ff..bfbf7b01f1ed 100644
--- a/arch/arm64/boot/dts/amlogic/overlays/odroidn2/Makefile
+++ b/arch/arm64/boot/dts/amlogic/overlays/odroidn2/Makefile
@@ -4,6 +4,7 @@ dtbo-y += \
 	spi0.dtbo \
 	pwm_cd.dtbo \
 	pwm_ef.dtbo \
+	pwmfan-lowtemp.dtbo \
 	uart0.dtbo \
 	uart1.dtbo \
 	sx865x-i2c1.dtbo \
diff --git a/arch/arm64/boot/dts/amlogic/overlays/odroidn2/pwmfan-lowtemp.dts b/arch/arm64/boot/dts/amlogic/overlays/odroidn2/pwmfan-lowtemp.dts
new file mode 100644
index 000000000000..bfa41cb90395
--- /dev/null
+++ b/arch/arm64/boot/dts/amlogic/overlays/odroidn2/pwmfan-lowtemp.dts
@@ -0,0 +1,20 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	fragement@0 {
+		target = <&fan_0>;
+
+		__overlay__ {
+			temperature = <45000>;
+		};
+	};
+
+	fragement@1 {
+		target = <&fan_1>;
+
+		__overlay__ {
+			temperature = <55000>;
+		};
+	};
+};
-- 
2.37.3

